ui/tests/acceptance/task-detail-test.js

```diff
@@ -37,32 +38,37 @@ test('/allocation/:id/:task_name should name the task and list high-level task i
 
 test('breadcrumbs includes allocations and link to the allocation detail page', function(assert) {
   assert.equal(
-    find('[data-test-breadcrumb="allocations"]').textContent.trim(),
-    'Allocations',
-    'Allocations is the first breadcrumb'
+    find('[data-test-breadcrumb="jobs"]').textContent.trim(),
+    'Jobs',
+    'Jobs is in the breadcrumb trail'
   );
   assert.equal(
-    find('[data-test-breadcrumb="allocations"]').getAttribute('href'),
-    '#',
-    "Allocations breadcrumb doesn't link anywhere"
+    find('[data-test-breadcrumb="jobs.job"]').textContent.trim(),
+    job.name,
+    'Job name is in the breadcrumb trail'
   );
   assert.equal(
-    find('[data-test-breadcrumb="allocation"]').textContent.trim(),
+    find('[data-test-breadcrumb="jobs.job.task-group"]').textContent.trim(),
+    allocation.taskGroup, // not taskGroupName ?
+    'Task Group is in the breadcrumb trail'
+  );
+  assert.equal(
+    find('[data-test-breadcrumb="allocations.allocation"]').textContent.trim(),
     allocation.id.split('-')[0],
-    'Allocation short id is the second breadcrumb'
+    'Allocation short id is in the breadcrumb trail'
```

Comment:
The change of language here just made me realize that the switch to `ember-test-selectors` removed the part of these tests that assert breadcrumb order. I think testing that is still important, and I'd love to hear your thoughts on the best way to go about doing that.